qt/aqt/sync.py

```diff
@@ -209,11 +209,30 @@ def on_full_sync_timer(mw: aqt.main.AnkiQt, label: str) -> None:
         return
     sync_progress = progress.full_sync
 
+    # If we've reached total, show the "checking" label
     if sync_progress.transferred == sync_progress.total:
         label = tr.sync_checking()
+
+    INT32_MAX = 2_147_483_647
```

Comment:
Sorry, forgot to mention that last time, but we can simplify the fix by just dividing `transferred` and `total` by 1024 and passing that to `mw.progress.update()`?